app/models/data_importer.rb

```diff
@@ -0,0 +1,35 @@
+require 'json'
+require "sinatra/activerecord"
+require_relative './well_inspection'
+
+class DataImporter
+
+  def init
+    save_objects
+  end
+
+  # for now get from data.json
+  # eventually hit the API endpoint and
+  # 1. parse headers to create table
+  # 2. build migration
+  # 3. run migration
+  # 4. parse json for data array
+  # 5. store json objs as ar instances
+  def get_data
+    null = nil
+
+    data = File.read('/home/alexa/Desktop/projects/denton_data/data/data.json')
```

Comment:
I plan to getting around to a lot of clean up tomorrow. :grin: I basically circumvented the existing app just to get AR up and running. Needs to be condensed back down.